provider/marathon/marathon_test.go

```diff
@@ -257,6 +257,124 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "multiple ports",
```

Comment:
There are also tests for frontend rules, etc..  But I'll add more cases specifically to `TestMarathonGetPort` one as well...